modules/rpc/yarpc.go

```diff
@@ -48,8 +52,9 @@ type YARPCModule struct {
 }
 
 var (
-	_dispatcherFn = defaultYARPCDispatcher
 	_dispatcherMu sync.Mutex
+	_dispatcherFn = defaultYARPCDispatcher
```

Comment:
maybe we should start commenting these? we've got a lot of globals to keep track of :(